### DISCUSSION TITLE
Welcome to Snippify-Hub Discussions!

### DISCUSSION THREAD

**Discussion by shrinidhihegde3** [shrinidhihegde3/Snippify-Hub#57] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yashksaini-coder** [comment]
Hi @shrinidhihegde3 is there a discord channel of yours. I would like to join to discuss some code changes in the project.

**Reply by shrinidhihegde3** [reply]
There is no particular channel is present but you can contact me directly here my discord username "shrinidhi_hegde"

**Reply by yashksaini-coder** [reply]
okh, sent you request on your discord. Please accept it

**Comment by shrinidhihegde3** [comment]
Any updates on the issue assigned?

**Reply by yashksaini-coder** [reply]
yes, but I kept getting a c98 error build dependency error. currently resolving it now. it is a local system error.

seems my laptop can't handle multiple editors being opened

**Comment by shrinidhihegde3** [comment]
Hello everyone, some of you haven't completed the assigned issues yet. I understand there was a festival recently, but now you can start contributing again...🐱🐱
